src/raw.js

```diff
@@ -136,6 +140,10 @@ function replaceKeyBindings(raw) {
     return full.replace(key, '?')
   })
 
+  emptyBindings.map(function(binding) {
+    sql = sql.replace(new RegExp(` ${binding}|${binding} |${binding}`, 'g'), '');
```

Comment:
@wubzz it should be enough that named bindings are converted to positional bindings with as little magic as possible and then it is job of some other part to decide how to handle `undefined` in positional binding.

I agree that replacing `undefined` with `DEFAULT` is not good functionality... probably throwing an error would be the best way to handle it for positional bindings and for named binding. Converting it to `DEFAULT` is just an accidental undefined behavior... Anyways discussion how to handle `undefined` in case of positional bindings should be done in separate pull request.